app/submitter/converter.py

```diff
@@ -4,6 +4,8 @@
 from datetime import datetime, timezone
 
 from app import settings
+from app.questionnaire_state.state_repeating_answer_question import _extract_answer_instance_id
```

Comment:
the underscore at the start of _extract_answer_instance_id implies that this is a "private" function?